src/styles/custom.css

```
@@ -53,7 +55,26 @@ body {
   color: var(--sl-color-accent-high);
 }
 
+/* プライマリーボタンのコントラスト比を確保する */
+.sl-link-button.primary {
+  font-weight: bold;
+}
+
 /* ナビゲーションパネルのリンクのコントラスト比を確保する */
 .right-sidebar-panel a[aria-current="true"] {
   color: var(--sl-color-accent-high);
 }
+
+[aria-current="page"] {
+  color: var(--sl-color-black);
+}
```

[nits]

ホバーすると文字色が変わってしまっていたため、ホバー時も同じカラーにすると良さそうです！

```suggestion
[aria-current="page"],
[aria-current="page"]:hover {
  color: var(--sl-color-black);
}
```